PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Clayton Kehoe <118750525+kehoecj@users.noreply.github.com>
+# Author : wiz64 <wiz64.com>
+pkgname=config-file-validator
+pkgver=1.4.0
+pkgrel=1
+pkgdesc="A tool to validate the syntax of configuration files"
+arch=('x86_64')
+url="https://github.com/Boeing/config-file-validator"
+license=('Apache 2.0')
+depends=('glibc')
+makedepends=('go')
+source=("$pkgname-$pkgver.tar.gz::$url/archive/v$pkgver.tar.gz")
+sha256sums=('SKIP')
+
+build() {
+  cd "$pkgname-$pkgver"
+  CGO_ENABLED=0 \
+  GOOS=linux \
+  GOARCH=amd64 \
+  go build \
+  -ldflags='-w -s -extldflags "-static"' \
+  -tags netgo \
+  -o config-file-validator \
```

Comment:
We've been building and publishing the executable as `validator` so that the name isn't quite so long to type on the command line. Did you find that conflicted with existing package names?

Reply:
Agree. Would like to keep executable name the same.

Reply:
> We've been building and publishing the executable as `validator` so that the name isn't quite so long to type on the command line. Did you find that conflicted with existing package names?

As you wish

Reply:
> We've been building and publishing the executable as `validator` so that the name isn't quite so long to type on the command line. Did you find that conflicted with existing package names?

I felt it makes more sense to use the full name.. but as you wish

Reply:
Yeah, I've been wondering if the binary needs to be named something less generic. I'd like to change it all at once in all places though so at least for this PR I think the status quo makes sense. Thanks!